server/src/main/java/org/tctalent/server/api/admin/CandidateBuilderSelector.java

```diff
@@ -92,15 +101,37 @@ public DtoBuilder selectBuilder() {
         User user = userService.getLoggedInUser();
         Partner partner = user == null ? null : user.getPartner();
 
+
+        //If partner is a job creator, look up the candidates for whom they can see all properties
+        //(ie candidates who have got past the CV Review stage in any job opps managed by the partner)
+
+        //Start by fetching all the opps associated with the partner
+        List<CandidateOpportunity> partnerOpps = candidateOpportunityService.findJobCreatorPartnerOpps(partner);
+
```

Comment:
Could this potentially become a very large list?

Reply:
Yes - it could get quite big - but never the whole database or anything like that. 

I could work with pages.